src/test/context.test.ts

```diff
@@ -36,7 +36,7 @@ describe('Context object', () => {
     expect(Context.HOST_NAME).toEqual(validParams.HOST_NAME);
   });
 
-  it("can't initialize with empty values", () => {
+  it('can\'t initialize with empty values', () => {
```

Comment:
I thought double quotes were allowed in this scenario? Did this fail with the rule?